hack/e2e/ebs.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -uo pipefail
+
+BASE_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+source "${BASE_DIR}"/util.sh
+
+function ebs_check_migration() {
+  loudecho "Checking migration"
+  # There should have been no calls to the in-tree driver kubernetes.io/aws-ebs but many calls to ebs.csi.aws.com
+  # Find the controller-manager log and read its metrics to verify
+  NODE=$(kubectl get node -l kubernetes.io/role=master -o json | jq -r ".items[].metadata.name")
+  kubectl port-forward kube-controller-manager-"${NODE}" 10252:10252 -n kube-system &
+
+  # Ensure port forwarding succeeded
+  while true; do
```

Comment:
Can we change this to give up after a while? Isn't this going to loop forever if forwarding fails?

Reply:
fixed, see commit "Time out if port forward fails"

waiting for tests to ppass, the snapshot ones failed because I didn't install the snapshot CRD, hence commit " Deploy snapshot CRD "